pkg/pool-manager/mac-pool-map.go

```diff
@@ -0,0 +1,118 @@
+package pool_manager
+
+import (
+	kubevirt "kubevirt.io/client-go/api/v1"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// isMacPendingCommit checks mac entry if an update is needed
+func (m macMap) isMacPendingCommit(macAddress string) bool {
+	macEntry, exist := m.findByMacAddress(macAddress)
+	if exist {
+		if macEntry.isPending() {
+			return true
+		}
+	}
+	return false
+}
+
+// clearMacTransactionFromMacEntry clears mac entry's transactionTimestamp, signalling that no further update is needed
+func (m *macMap) clearMacTransactionFromMacEntry(macAddress string) {
+	macEntry, exist := m.findByMacAddress(macAddress)
+	if exist {
+		(*m)[macAddress] = macEntry.resetTimestamp()
+	}
+}
+
+func (m macMap) findByMacAddress(macAddress string) (macEntry, bool) {
+	macEntry, exist := m[macAddress]
+	return macEntry, exist
```

Comment:
`return m[macAddress]`

Reply:
it doesn't compile from some reason. but I also prefer it for readablity